lang/semgrep-grammars/src/semgrep-elixir/grammar.js

```diff
@@ -46,10 +55,12 @@ module.exports = grammar(base_grammar, {
     pair: ($, previous) => {
       return prec(-1, choice(
         previous,
-        '...',
+        $.semgrep_ellipsis,
       ));
     },
 
+    semgrep_ellipsis: $ => prec(-1, '...'),
```

Comment:
why the prec? would be good to give the intuition on why we need this directive, like giving an example
with a conflict and how prec(-1 does the right thing and gives priority to one or the other to generate the right AST.
You can look at what we do in the other semgrep-xxx/grammar.js, we sometimes explain those prec and conflicts.

Reply:
Added comments to explain why we need prec and why its correct. Thanks!